webdriver/webapp_server.go

```diff
@@ -28,7 +29,7 @@ var (
 )
 
 // StaticTestDataRevision is the SHA for the local (static) test run summaries.
-const StaticTestDataRevision = "b952881825"
+const StaticTestDataRevision = "b952881825e7d3974f5c513e13e544d525c0a631"
```

Comment:
If you change this variable, you'll need to either fix this line:
https://github.com/web-platform-tests/wpt.fyi/blob/c6886c69ba71795f11be1feb71ec4c58185feda2/webdriver/webapp_server.go#L252

or rename the static directory.

Reply:
Done.